PowerNSX.psm1

```diff
@@ -22026,12 +22043,18 @@ function New-NsxFirewallSection  {
         [System.XML.XMLElement]$xmlRoot = $XMLDoc.CreateElement("section")
         $xmlDoc.appendChild($xmlRoot) | out-null
 
+        #Mandatory Fields
         Add-XmlElement -xmlRoot $xmlRoot -xmlElementName "name" -xmlElementText $Name
 
+        #Optional Fields
+        if ( $PsBoundParameters.ContainsKey('Universal')) {
```

Comment:
I just modified this and ran it through the test environment and it created the rulesets without the extra $PSBoundParameters.